### DISCUSSION TITLE
[Feature] Put image filename in search index

### DISCUSSION THREAD

**Discussion by erikvanoosten** [immich-app/immich#5191] [Feature Request]
### The feature

I just imported 20k photo's into Immich. Many of those have relevant names in their folder and filename. Please consider adding these to the search index.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by alextran1502** [comment]
You can already search file name with `m:<file-name>`

**Comment by chengguangnan** [comment]
I'm using v1.91.4 and `m:<file-name>` returns no results, meanwhile `m:<city>` works correctly.

It's been mentioned in https://github.com/immich-app/immich/issues/5747 by `jkumeboshi`.

Also, try it at current demo site `https://demo.immich.app/search?q=Walls_of_Dubrovnik` retuns no results.

**Reply by undaunt** [reply]
Same here, I'm trying to find a specific file by file name and cannot do it. 1.92.1.

**Reply by AndyPro720** [reply]
Problem still persists in 1.97.0, will have look at ease.
